### PR TITLE
Update Issue and Pull Request templates.

### PR DESCRIPTION
**Pull request by cheenamalhotra**
Guide Articles : 
- [Creating Issue Templates](https://help.github.com/articles/creating-issue-templates-for-your-repository/)
- [Creating PR Templates](https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+name: Feature Request
+about: Suggest an idea for this project
+title: "[FEATURE REQUEST]"
+labels: enhancement
+assignees: ''
+
+---
+
+## Is your feature request related to a problem? Please describe.
```
Comment (rene-ye) [inline .github/ISSUE_TEMPLATE/feature-request.md]: If so, please give a short summary of the problem and how the feature would resolve the problem.

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+<!--- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] --->
+
+## Describe the solution you'd like
```
Comment (rene-ye) [inline .github/ISSUE_TEMPLATE/feature-request.md]: Describe the preferred solution.

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+---
+
+## Is your feature request related to a problem? If so, please give a short summary of the problem and how the feature would resolve it.
+<!--- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] --->
+
+## Describe the preferred solution.
+<!--- A clear and concise description of what you want to happen. --->
+
+## Describe alternatives you've considered
```
Comment (rene-ye) [inline .github/ISSUE_TEMPLATE/feature-request.md]: Should this go under optional?

File: .github/PULL_REQUEST_TEMPLATE/bug_fix.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+name: Bug Fix
+about: Fix for a driver issue
+title: "Fix | "
+
+---
+
+## Problem description
+<!--- Provide full details of the problem if no corresponding GitHub issue exists. --->
+
+### Fixes existing GitHub issue :
```
Comment (rene-ye) [inline .github/PULL_REQUEST_TEMPLATE/bug_fix.md]: Only line with " : "

File: .github/PULL_REQUEST_TEMPLATE/feature.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+about: Propose a new feature to the driver
+title: "Feature | "
+
+---
+
+## Brief Description
+<!--- Briefly describe the feature introduced with the PR. --->
+
+<!--- Optional --->
+### Fixes existing GitHub issue :
```
Comment (rene-ye) [inline .github/PULL_REQUEST_TEMPLATE/feature.md]: Only line with " : "

File: .github/PULL_REQUEST_TEMPLATE/other.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+about: Propose a custom change to the project source code.
+title: "Other | "
+
+---
+
+## Brief Description
+<!--- Briefly describe the feature introduced with the PR. --->
+
+<!--- Optional --->
+### Fixes existing GitHub issue :
```
Comment (rene-ye) [inline .github/PULL_REQUEST_TEMPLATE/other.md]: Only line with " : "

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+title: "[FEATURE REQUEST]"
+labels: enhancement
+assignees: ''
+
+---
+
+## Is your feature request related to a problem? If so, please give a short summary of the problem and how the feature would resolve it.
+<!--- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] --->
+
+## Describe the preferred solution.
```
Comment (rene-ye) [inline .github/ISSUE_TEMPLATE/feature-request.md]: Use of period here is inconsistent.

File: .github/ISSUE_TEMPLATE/feature-request.md (hunk 3 above)
Comment (cheenamalhotra) [reply .github/ISSUE_TEMPLATE/feature-request.md]: That ways everything is optional.. but knowing alternatives kind of tells us where the idea is coming from and is more probable to mention... so I'll keep it as it is.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+## Problem description
+<!--- Provide full details of the problem. -->
+1. Expected behaviour:
+2. Actual behaviour:
+3. Error message/stack trace:
+4. Any other details that can be helpful:
+
+## JDBC trace logs
+<!--- Provide the JDBC trace logs. Instructions can be found here: https://docs.microsoft.com/en-us/sql/connect/jdbc/tracing-driver-operation?view=sql-server-2017 -->
```
Comment (peterbae) [inline .github/ISSUE_TEMPLATE/bug_report.md]: remove the en-us in the URL

Comment (peterbae) [reply .github/ISSUE_TEMPLATE/bug_report.md]: other than that the PR looks good

**Pull request merged**